persistence/account.go

```diff
@@ -18,11 +17,13 @@ func (p PostgresContext) GetAccountAmount(address []byte, height int64) (amount
 }
 
 func (p PostgresContext) getAccountAmountStr(address string, height int64) (amount string, err error) {
-	ctx, conn, err := p.GetCtxAndConnection()
+	ctx, txn, err := p.DB.GetCtxAndTxn()
 	if err != nil {
 		return
 	}
-	if err = conn.QueryRow(ctx, schema.GetAccountAmountQuery(address, height)).Scan(&amount); err != nil {
+	amount = "0"
```

Comment:
```
We only return this default value if err != nil, but if err != nil, then the result should not be used because we should handle the error.
```
The error returned can be `row_not_found` (pseudo) which should be translated to the default value

Reply:
We can handle this higher in the stack if you prefer